src/lib/util.ts

```diff
@@ -720,7 +720,6 @@ export function removeFromArr<T>(arr: T[], item: T) {
  * @param percent The percent to scale
  * @returns percent
  */
-export function exponentialPercentScale(percent: number, decay = 0.0038) {
-	const decayedPercent = Math.pow(100 * Math.E, -decay * (100 - percent));
-	return decayedPercent * 100;
+export function exponentialPercentScale(percent: number, decay = 0.021) {
+	return 100 * Math.pow(Math.E, -decay * (100 - percent));
```

Comment:
this gonna break/change the current scaling though? :S

Reply:
No it's a drop in replacement.

The numbers are off by a very small margin, but nothing noticeable. (1% average)

I did a lot of testing to make sure it wouldn't break anything, you put `90` into each function, one gives 80.8 one gives 81.05